<!--REQUEST_ID=43c8bcee-3b96-4c01-982c-36cfbdf4780a-->
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.50247&attempt_id=98885e35-39f6-4bb8-9f42-addd5c9bd265)

Attempted: jxrlib

<details><summary>Partial log (click to expand)</summary><p>

```
      _ParseTifDEValue in JXRTestTif.o
  "_WMPFree", referenced from:
      _PKImageDecode_Release_TIF in JXRTestTif.o
  "_WMPhotoTranscode", referenced from:
      _PKImageEncode_Transcode_WMP in JXRGlueJxr.o
ld: symbol(s) not found for architecture x86_64
clang-5.0: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [Makefile:166: /private/tmp/nix-build-jxrlib-1.1.drv-0/source/build/libjxrglue.so] Error 1
builder for '/nix/store/54kd7rz117sqrhpya57pcczmzmlkzm7y-jxrlib-1.1.drv' failed with exit code 2
error: build of '/nix/store/54kd7rz117sqrhpya57pcczmzmlkzm7y-jxrlib-1.1.drv' failed
```
</p></details>

--- a/pkgs/development/libraries/jxrlib/default.nix
+++ b/pkgs/development/libraries/jxrlib/default.nix
@@ -1,0 +1,28 @@
+{ stdenv, fetchFromGitHub, python }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "jxrlib";
+  version = "1.1";
+
+  # Use the source from a fork on github because CodePlex does not
+  # deliver an easily downloadable tarball.
+  src = fetchFromGitHub {
+    owner = "4creators";
+    repo = pname;
+    rev = "f7521879862b9085318e814c6157490dd9dbbdb4";
+    sha256 = "0rk3hbh00nw0wgbfbqk1szrlfg3yq7w6ar16napww3nrlm9cj65w";
+  };
+
+  nativeBuildInputs = [ python ];
+
+  makeFlags = [ "DIR_INSTALL=$(out)" "SHARED=1" ];
+
+  meta = with stdenv.lib; {
+    description = "Implementation of the JPEG XR image codec standard";
+    homepage = https://jxrlib.codeplex.com;
+    license = licenses.bsd2;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ romildo ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -10261,6 +10261,8 @@ with pkgs;
 
   jsonrpc-glib = callPackage ../development/libraries/jsonrpc-glib { };
 
+  jxrlib = callPackage ../development/libraries/jxrlib { };
+
   libjson = callPackage ../development/libraries/libjson { };
 
   libb64 = callPackage ../development/libraries/libb64 { };

